client/public/index.html

```diff
@@ -9,6 +9,10 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
this could belong to another pull request :)

Reply:
Thank you. I know 🙈